src/renderer/drawers/OperationDetails/index.js

```diff
@@ -124,6 +137,10 @@ const OperationD: React$ComponentType<Props> = (props: Props) => {
   const { extra, hash, date, senders, type, fee, recipients: _recipients } = operation;
   const recipients = _recipients.filter(Boolean);
   const { name } = mainAccount;
+  const operations = useMemo(() => [operation], [operation]);
+  const nfts = nftsFromOperations(operations);
+  const { status, metadata } = useNftMetadata(nfts[0]?.collection.contract, nfts[0]?.tokenId);
```

Comment:
same here seems counterintuitive to get only the first element of nftsFromOperations it should return that element directly if you're only using/expect one

Reply:
That's some of the limitation we have because the backend lacks informations...
The general idea is that an NFT collection should have an image, but right now it's not the case. So we are using the first nft of the collection as the image of that collection (+ it contains also the name of the collection).

That's some architecture issue that we can't really resolve right now. 😞